src/test/java/Homework17.java

```diff
@@ -12,7 +12,7 @@ public class Homework17 extends BaseTest {
     public void addSongToPlaylist() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

**Thread. sleep is bad!** It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.

![Screenshot_1](https://user-images.githubusercontent.com/104761482/225236947-079c0972-3177-4a0c-b28b-3e077ceb7a28.jpg)